flyway/scxa/migrations/V18__scxa-create-exp-design.sql

```diff
@@ -0,0 +1,32 @@
+CREATE TABLE exp_design
+(
+    accession VARCHAR(255)
+        CONSTRAINT exp_design_k
+            REFERENCES experiment (accession)
+            ON DELETE CASCADE,
+    marray VARCHAR(255),
+    sample VARCHAR(255),
+    annot_type char(50)
```

Comment:
Do we really need to store the strings 'factor' and 'characteristic'? Seems like this could just be a boolean column called 'factor', which would be massively more efficient.

Reply:
That's a good one! makes sense will convert it to a boolean one